applications/matter_bridge/doc/adding_bridged_matter_device.rst

```diff
@@ -0,0 +1,397 @@
+.. _matter_adding_bridged_matter_device:
+
+Adding support for bridging of new Matter device type
+*****************************************************
+
+The Matter Bridge application supports bridging only a few Matter device types due to practical reasons.
+You can select any of the available :ref:`Matter device types <ug_matter_device_types>` and add support to it in the application.
+In this guide the ``Pressure Sensor`` device type was presented as an example.
+
+Based on :ref:`Matter Bridge architecture <ug_matter_overview_bridge_ncs_implementation>` you will need to implement the ``Matter Bridged Device`` and ``Bridged Device Data Provider`` roles for the newly added Matter device type.
+The Matter Bridge application supports simulated and Bluetooth LE Bridged Device Data Providers.
+In this guide the simulated provider example was presented, but the process looks similar for Bluetooth LE provider as well.
+
+Complete the following steps to add the ``Pressure Sensor`` device type support to the Matter Bridge.
+
+1. Enable ``Pressure Measurement`` cluster for the endpoint ``2`` in the :file:`src/bridge.zap` file and re-generate files located in the :file:`src/zap-generated` directory.
+   To learn how to modify the :file:`.zap` file and re-generate the :file:`zap-generated` directory, see the :ref:`ug_matter_creating_accessory_edit_zap` in the :ref:`ug_matter_creating_accessory` user guide.
+#. Implement the ``Matter Bridged Device`` role.
+
+   a. Create :file:`pressure_sensor.cpp` and :file:`pressure_sensor.h` files in the :file:`src/bridged_device_types` directory.
+   #. Open the :file:`nrf/samples/matter/common/bridge/matter_bridged_device.h` to see :c:struct:`MatterBridgedDevice` class constructor.
```

Comment:
This step is needed to learn the abstract class constructor signature, that is used in the child class constructor, right? Maybe it's worth clarifying?

Reply:
Added one sentence clarifying it.